test/unit/api/utils.spec.js

```diff
@@ -134,20 +154,166 @@ describe('Utilities', () => {
       const randomTextNonAscii = 'simple text \n = ascii';
       const randomTextMarkdown = 'simple text \n # markdown';
 
-      expect(parseReadme('testPackage', randomText)).toEqual('<p>%%%%%**##==</p>\n');
-      expect(parseReadme('testPackage', simpleText)).toEqual('<p>simple text</p>\n');
-      expect(parseReadme('testPackage', randomTextNonAscii))
-        .toEqual('<p>simple text \n = ascii</p>\n');
-      expect(parseReadme('testPackage', randomTextMarkdown))
-        .toEqual('<p>simple text </p>\n<h1 id="markdown">markdown</h1>\n');
+      expect(parseReadme('testPackage', randomText)).toEqual(
+        '<p>%%%%%**##==</p>\n'
+      );
+      expect(parseReadme('testPackage', simpleText)).toEqual(
+        '<p>simple text</p>\n'
+      );
+      expect(parseReadme('testPackage', randomTextNonAscii)).toEqual(
+        '<p>simple text \n = ascii</p>\n'
+      );
+      expect(parseReadme('testPackage', randomTextMarkdown)).toEqual(
+        '<p>simple text </p>\n<h1 id="markdown">markdown</h1>\n'
+      );
     });
 
     test('should show error for no readme data', () => {
       const noData = '';
-      const spy = jest.spyOn(Logger.logger, 'error')
-      expect(parseReadme('testPackage', noData))
-        .toEqual('<p>ERROR: No README data found!</p>\n');
-      expect(spy).toHaveBeenCalledWith({'packageName': 'testPackage'}, '@{packageName}: No readme found');
+      const spy = jest.spyOn(Logger.logger, 'error');
+      expect(parseReadme('testPackage', noData)).toEqual(
+        '<p>ERROR: No README data found!</p>\n'
+      );
+      expect(spy).toHaveBeenCalledWith(
+        { packageName: 'testPackage' },
+        '@{packageName}: No readme found'
+      );
+    });
+  });
+
+  describe('addGravatarSupport', () => {
+    test('check for blank object', () => {
+      expect(addGravatarSupport({})).toEqual({});
+    });
+    test('author field', () => {
+      const pkgWithAuthorInfo = {
```

Comment:
I'd rather split these 3 scenarios in different `test()` blocks. Easier for maintenance. As smaller is the unit test, easier is to update.